packages/atlas-experiment-table/src/TableCell.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import URI from 'urijs'
+
+import { Table, Text } from 'evergreen-ui'
+
+const TableCell = ({ dataRow, dataKey, image, linkTo, host, width }) => {
+  // If the column is an image put the image in the cell (or an unknown icon if the type is undefined)
```

Comment:
Personally, if the `if-else` logic level goes too deep (>2), I will use `if-else` or `switch` to make the code easy to be followed. 

Otherwise, line 8 comment is better to be moved below `const cellItem = image ?` line.

Reply:
Something like this?

```javascript
  let cellItem = null
  if (image) {
    // If the column is an image put the image in the cell (or an unknown icon if the type is undefined)
    if (image[dataRow[dataKey]]) {
      cellItem = <img {...image[dataRow[dataKey]]}/>
    }
    else {
      cellItem = `❔`
    }
  } else if (Array.isArray(dataRow[dataKey])) {
    // If the contents of the cell is an array expand it to a list
    cellItem =
      <ul>
        {dataRow[dataKey].map(
          (element, index) =>
            <li key={`${index}`}>
              {element}
            </li>
        )}
      </ul>
  } else {
    // Any other type (i.e. string, number) put it as-is in the cell
    cellItem = dataRow[dataKey]
  }
```

Reply:
As a general rule I tend to favour the ternary operator because it returns a value (it’s an expression) instead of `if-else` which is a control flow statement. The ternary operator is thus more declarative and you can assign it to a `const`, whereas with `if` you need to declare your variable with `let` and your code is more imperative.

In any case, I can understand that in this case regular conditionals can make the code more readable and I changed the logic to use `if-else`. Here we can’t use `switch` because the expression under evaluation is not the same at each conditional. Maybe if JavaScript had [pattern matching](https://en.wikipedia.org/wiki/Pattern_matching)... [One can dream, though!](https://github.com/bramstein/funcy)